integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -32,17 +32,17 @@
 
     requires java.logging;
 
-    requires static jakarta.annotation;
+    requires jakarta.annotation;
 
     requires io.helidon.config;
-    requires io.helidon.common.http;
-    requires io.helidon.nima.servicecommon;
+    requires transitive io.helidon.common.http;
+    requires transitive io.helidon.nima.servicecommon;
```

Comment:
Again, should not be transitive, as the API does not expose classes from this module

Reply:
how about `io.helidon.nima.servicecommon.HelidonFeatureSupport` as `MicrometerFeature` extends it?